<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSetFreeRDP</class>
 <widget class="QDialog" name="CDlgSetFreeRDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set FreeRdp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../App/Client/Resource/Resource.qrc">
    <normaloff>:/image/App</normaloff>:/image/App</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel(&amp;C)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Server">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePassword">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbShow">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../App/Client/Resource/Resource.qrc">
             <normaloff>:/image/EyeOn</normaloff>:/image/EyeOn</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>User name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leUserName"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="cbSavePassword">
           <property name="text">
            <string>Save password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbOnlyView">
           <property name="text">
            <string>Only View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Domain:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leDomain"/>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="cbClipboard">
           <property name="text">
            <string>Enable clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Reconnect interval:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbReconnect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0: don't reconnect&lt;/p&gt;&lt;p&gt;&amp;gt;0: reconnect interval&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="statusTip">
            <string>Reconnect interval</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0: don't reconnect&lt;/p&gt;&lt;p&gt;&amp;gt;0: reconnect interval&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Server:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leServer">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spPort">
           <property name="maximum">
            <number>65536</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leName">
           <property name="toolTip">
            <string>Connecter name</string>
           </property>
           <property name="statusTip">
            <string>Connecter name</string>
           </property>
           <property name="whatsThis">
            <string>Connecter name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="cbShowServerName">
         <property name="text">
          <string>Show server name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Remote desktop size:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="rbFullScreen">
              <property name="text">
               <string>Full screen</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QRadioButton" name="rbSelect">
              <property name="text">
               <string>Select size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbDesktopSize"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cbAllMonitor">
            <property name="text">
             <string>Use all local monitors for remote sessions</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Remote color depth:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbColorDepth"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>131</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leDomain</tabstop>
  <tabstop>leServer</tabstop>
  <tabstop>spPort</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>cbSavePassword</tabstop>
  <tabstop>cbOnlyView</tabstop>
  <tabstop>cbClipboard</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>rbFullScreen</tabstop>
  <tabstop>rbSelect</tabstop>
  <tabstop>cbDesktopSize</tabstop>
  <tabstop>cbAllMonitor</tabstop>
  <tabstop>cbColorDepth</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../App/Client/Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
